x/compute/genesis_test.go

```diff
@@ -113,11 +113,11 @@ func TestInitGenesis(t *testing.T) {
 
 	assertContractList(t, q, data.ctx, 1, []string{contractAddr.String()})
 	assertContractInfo(t, q, data.ctx, contractAddr, 1, creator)
-	assertContractState(t, q, data.ctx, contractAddr, state{
-		Verifier:    []byte(fred),
-		Beneficiary: []byte(bob),
-		Funder:      []byte(creator),
-	})
+	// assertContractState(t, q, data.ctx, contractAddr, state{
+	// 	Verifier:    []byte(fred),
+	// 	Beneficiary: []byte(bob),
+	// 	Funder:      []byte(creator),
+	// })
```

Comment:
why did you comment this out?
also other pieces of tests?

Reply:
All the commented out tests are testing the query functions we removed e.g. query state raw, etc.. We have no use for them as raw state is ecrypted